src/api/userComments.jsx

```diff
@@ -0,0 +1,18 @@
+import * as firebase from 'firebase';
+import 'firebase/firestore';
+
+// This function will add a comment to the DB specified by ID
+export const addUserComment = async (comment, id) => {
```

Comment:
With this comments collection, it is impossible to match comments to ratings. Would it make more sense to save the comment along with the rating in the `addBikeRating` function (so in the ratings collection)? Then we could get a list that displays the rating and the comment. Do we want to save the username as well, or keep all comments and ratings anonymous?

Reply:
In order to keep things simple, I'd suggest keeping the comments/ratings anon.